test/python/WMComponent_t/DBS3Buffer_t/DBSBufferFile_t.py

```diff
@@ -947,7 +947,7 @@ def testProperties(self):
                                configContent="MOREGIBBERISH")
         testFileA.setDatasetPath("/Cosmics/CRUZET09-PromptReco-v1/RECO")
         testFileA.setValidStatus(validStatus="VALID")
-        testFileA.setProcessingVer(ver="ProcVer")
+        testFileA.setProcessingVer(ver="123")
```

Comment:
shouldn't it be integer?

Reply:
I think we need to test code which should handle string input since there is no data-type checking anywhere in a code and moreover we know that different types can be passed to the code.